core/src/main/scala/zio/config/ConfigStringModule.scala

```diff
@@ -382,7 +384,7 @@ trait ConfigStringModule extends ConfigModule with ConfigSourceStringModule {
       configDescriptor: ConfigDescriptor[A],
       keyDelimiter: Option[Char] = None,
       valueDelimiter: Option[Char] = None
-    )(implicit tag: Tag[A]): Layer[ReadError[String], Config[A]] =
+    )(implicit tag: Tag[A]): ZLayer[SystemModule, ReadError[String], Config[A]] =
```

Comment:
This is minor, but worth noting. All other functions in this module returns a Layer.
That said, I realise myself that not all functions in `ConfigSourceModule` returned `IO` before.

Reply:
hmm you want me to change return types for others to ZLayer[Any then ? :)

Reply:
btw not all:

 ```
def fromSystemProperties[K, V, A](
      configDescriptor: ConfigDescriptor[A],
      keyDelimiter: Option[Char] = None,
      valueDelimiter: Option[Char] = None
    )(implicit tag: Tag[A]): ZLayer[System, ReadError[String], ZConfig[A]]
```
is returning ZLayer here already